bluebrain/repo-bluebrain/packages/py-bglibpy/package.py

```diff
@@ -13,17 +13,17 @@ class PyBglibpy(PythonPackage):
 
     version('develop', branch='main')
 
-    version('4.4.36', commit='07fe9999a137c3741fd95713149a76a202cb7d7a')
+    version('4.7.2', commit='50e5e46b97105fb2b3fe67fba883d5c3c21e1134')
 
     depends_on('py-setuptools', type=('build', 'run'))
     depends_on('neuron+python', type='run')
+
+    # dependencies from setup.py
     depends_on('py-numpy@1.8:', type='run')
     depends_on('py-matplotlib@3.0.3:', type='run')
     depends_on('py-cachetools', type='run')
-    depends_on('py-h5py@2.3:', type='run')
-
-    depends_on('py-bluepy@2.1:2.999', type='run')
-    depends_on('py-libsonata', type='run')
+    depends_on('py-bluepy@2.4.2:2.999', type='run')
+    depends_on('py-bluepy-configfile@0.1.18:', type='run')
```

Comment:
I think this clashes with `py-minis-validation`, which requires both `py-bglibpy` and `py-bluepy-configfile@0.1.10:0.999`.

Reply:
But a pair of `py-bluepy` and `py-bluepy-configfile` versions that satisfy both `py-bglibpy` and `py-minis-validation` can be found I think.

`py-minis-validation` has
```
    depends_on('py-bluepy@2.0:2.999', type='run')
    depends_on('py-bluepy-configfile@0.1.10:0.999', type='run')
    depends_on('py-bglibpy@4.4:4.999', type='run')
```

`py-bglibpy` has

```
    depends_on('py-bluepy@2.4.2:2.999', type='run')
    depends_on('py-bluepy-configfile@0.1.18:', type='run')

```

e.g. using py-bluepy>=2.4.2,<2.999 and py-bluepy-configfile>=0.1.18,<0.999 will satisfy both packages.

Reply:
I just added an upperbound to bluepy-configfile `depends_on('py-bluepy-configfile@0.1.18:0.999', type='run')` to try.

Reply:
Adding the upperbound didn't fix it. On top of that it caused the `science` pipeline to fail.

Reply:
I reverted that commit. The error in the NSE pipeline is this

```
Error: py-minis-validation%gcc ^neuron%intel is unsatisfiable, errors are:
  no version satisfies the given constraints
```

Reply:
Probably the upperbound on py-minis-validation is to exclude the `develop` version. The develop version is the only version that satisties >0.1.18

Reply:
`py-bluepy-configfile>=0.1.18` is needed for the GluSynapse simulations.

@mgeplf @joni-herttuainen how should we do? Does it make sense to release a new `py-bluepy-configfile` or to release a `py-bluepy` that enforces `py-bluepy-configfile>=0.1.18`?

Reply:
@anilbey, sorry just saw your comments now (tab stale 🤦 ). Yes, we'd need a new version of `py-bluepy-configfile`!